client/utils/test/account-fees.tsx

```diff
@@ -265,4 +275,97 @@ describe( 'Account fees utility functions', () => {
 			);
 		} );
 	} );
+
+	describe( 'formatMethodFeesTooltip()', () => {
+		beforeAll( () => {
+			global.wcpaySettings = { connect: { country: 'US' } };
+		} );
+		afterAll( () => {
+			global.wcpaySettings = { connect: { country: '' } };
+		} );
+
+		it( 'displays base percentage and fixed fee - no custom fee nor discount', () => {
+			const methodFees = mockAccountFees( {
+				percentage_rate: 0.123,
+				fixed_rate: 456.78,
+				currency: 'USD',
+			} );
+
+			methodFees.additional = {
+				percentage_rate: 0.01,
+				fixed_rate: 0,
+				currency: 'USD',
+			};
+
+			methodFees.fx = {
+				percentage_rate: 0.01,
+				fixed_rate: 0,
+				currency: 'USD',
+			};
+
+			const renderMethodFeesTooltip = () => {
```

Comment:
I'd suggest a different approach here, that can help in avoiding the repetition of this function. I'd say to pass the result of `render( formatMethodFeesTooltip( methodFees ) )` directly to the expect function below. Unless I am missing something.

I also suggest taking a snapshot of the HTML only. We are currently taking it from the result of calling the render function, which includes other stuff. I'd say that we should only care about the `container` property of the [rendered](https://testing-library.com/docs/react-testing-library/api#container-1) result.

Reply:
Thank you for guiding me through these great improvements! I have made the changes here : `ca527b5`